### PR TITLE
Add post code

### PR DESCRIPTION
**Pull request by swimjiy**
### Done
- post code (`PUT /api/v1/post/:id`, `DELETE /api/v1/post/:id` )

### CODE REVIEW

File: src/services/post.service.ts [hunk 1]
```diff
@@ -13,3 +13,17 @@ export async function addPost(title: string, content: string) {
...
+export async function updatePost(id: number, title: string, content: string) {
+  const post: Post = await Post.findOne({ id });
+  post.title = title;
+  post.content = content;
+  await post.save();
+  return post;
+}
+
+export async function deletePost(id: number) {
+const post: Post = await Post.findOne({ id });
```
Comment (pengin7384) [inline src/services/post.service.ts]: const 앞에 2 공백 및 `{id}`가 적합할 거 같습니다! (ref:[google js style guide](https://google.github.io/styleguide/jsguide.html#features-objects-mixing-keys))

**Comment by pengin7384** [comment]
LGTM with a minor comment! 고생하셨습니다!!!

**Comment by god9599** [comment]
LGTM!!!

**Review by god9599 — APPROVED** [review]

File: src/services/post.service.ts (hunk 1 above)
Comment (swimjiy) [reply src/services/post.service.ts]: 넵 수정했습니다!🙏

**Pull request merged**